packages/edit-site/src/components/editor-canvas-container/index.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * WordPress dependencies
+ */
+import { Children, cloneElement, useState, useMemo } from '@wordpress/element';
+import { createSlotFill, Button } from '@wordpress/components';
+import { ESCAPE } from '@wordpress/keycodes';
+import { __ } from '@wordpress/i18n';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { closeSmall } from '@wordpress/icons';
+import { useFocusOnMount, useFocusReturn } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from '../../private-apis';
+import { store as editSiteStore } from '../../store';
+
+/**
+ * Returns a translated string for the title of the editor canvas container.
+ *
+ * @param {string} view Editor canvas container view.
+ *
+ * @return {string} Translated string corresponding to value of view. Default is ''.
+ */
+export function getEditorCanvasContainerTitle( view ) {
```

Comment:
Not sure where this should go. Oh the agony.

I want a central place where we can match the current value of `store.editorCanvasContainerView` to a translated string. 

Something like this is needed to get the string across the app, e.g.,  `header-edit-mode/index.js`

?

Reply:
Good question — personally I'd probably leave it in this module for now, since it's close to where it's being used? It could always be moved around later.